r/src/test/java/org/molgenis/r/service/ProcessServiceImplTest.java

```diff
@@ -56,9 +57,19 @@ void testTerminate() throws RServerException {
   }
 
   @Test
+  @Disabled
```

Comment:
why disable test?

Reply:
I did not manage to convert its Rock stub ... see FIXME comment below

Reply:
Discussed: we'll fix those after merging and think need to